nautobot_chatops/api/views/ms_teams.py

```diff
@@ -139,6 +139,7 @@ def post(self, request, *args, **kwargs):
             "user_id": body["from"]["id"],
             "user_name": body["from"]["name"],
             "user_role": body["from"].get("role"),
+            "user_ad_id": body["from"].get("aadObjectId"),
```

Comment:
Just want to make sure `aadObjectId` is correct here

Reply:
Yes it is